docker-compose/cassandra-3.11/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+
+services:
+  cassandra-1:
+    image: cassandra:${CASSTAG}
+    networks:
+      - stargate
+    ports:
+      - 9044:9042
+    mem_limit: 2G
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cassandra-1
```

Comment:
hmm is this correct, point to itself as the seed?

Reply:
confirmed this is not necessary, removing